include/trigger/Issues.hpp

```diff
@@ -189,6 +189,14 @@ ERS_DECLARE_ISSUE_BASE(trigger,
                        "Factory could not find requested item " << item << ".",
                        ((std::string)item),
                        ERS_EMPTY)
+
+ERS_DECLARE_ISSUE_BASE(trigger,
+                       ConfigurationProblem,
```

Comment:
Unless this will be used globally it should be renamed to more easily point to the issue. IE `UnalignedHeartbeat` is very precise/clear, etc...

Reply:
The idea is that it could be used more globally, as a general "there was an error" and "here's a message that dev wanted to include here". I can make it more precise to?

Reply:
I think the idea of the issue system was to be very specific (certainly the case for currently existing issues). Maybe if you want it to be used globally (for that plugin) add HSI/timing or some reference to the names?

Reply:
Changed, but only to move this forward -- I still don't like it.
I did have a look at other Issue.hpp, e.g. in appfwk and readout, and many of the issues are very generic there so they can be reused across the whole repository, rather than only in one class.
Adding things like names of classes is redundant -- this is going to be added automatically anyway by the logger.

Reply:
I think having a global 'IssueAtConfig' is better (with you on this), I'm just pointing out that this is not the case for the Issues we have currently available, and therefore - for consistency - we should apply the same method now until a full rework of issues is done (probably not best time to do now).